plugins/newsdownloader.koplugin/main.lua

```diff
@@ -315,4 +315,16 @@ function NewsDownloader:setCustomDownloadDirectory()
     }:chooseDir()
 end
 
+function NewsDownloader:onCloseDocument()
+    local document_path = self.ui.document._document.filename
```

Comment:
Since you only want the path use http://koreader.rocks/doc/modules/util.html#splitFilePathName